lib_eio/unix/eio_unix.mli

```diff
@@ -84,6 +84,20 @@ module Stdenv : sig
       You can use the functions in {!Eio.Stdenv} to access these fields if you prefer. *)
 end
 
+module Sockopt : sig
+  type 'a t = 'a Eio_unix__Sockopt.t =
```

Comment:
Why refer to the `Eio_unix__Sockopt` internal module here?

Reply:
Removed in latest patchset; all one extensible variant now.